shopfloor_mobile/static/wms/src/settings/settings.js

```diff
@@ -6,7 +6,28 @@
 
 export var SettingsControlPanel = Vue.component("settings-control-panel", {
     data: function() {
-        return {};
+        return {
+            // Why do I have to do the sub 1 to get the correct name ?
+            profile_name: this.$root.has_profile
+                ? this.$root.profiles[this.$root.profile.id - 1].name
+                : "?",
```

Comment:
why can't you use `this.$root.profile.name`?

Reply:
Doh, changed